src/pages/ctrader-signup/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { form_style } from './ctrader-form.module.scss'
+import Flex from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import { Localize, WithIntl } from 'components/localization'
+import Link from 'features/components/atoms/link'
+import useRegion from 'components/hooks/use-region'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+import CtraderWrapper from 'components/custom/_ctrader-wrapper'
+import SignUpCtraderForm from 'components/custom/_signup-ctrader-form'
+import Layout from 'features/components/templates/layout'
+
+const SignUpFormContainer = () => {
+    const { is_eu } = useRegion()
+
+    const { is_mobile_or_tablet } = useBreakpoints()
+
+    const security_pdf_link = `/tnc${is_eu ? '/eu' : ''}/security-and-privacy.pdf`
+
+    return (
+        <Layout>
+            <CtraderWrapper>
+                <Flex.Box className={form_style} direction="col" gap="8x">
+                    <Typography.Heading size={is_mobile_or_tablet ? 'small' : 'xs'}>
+                        <Localize translate_text="_t_Let's start_t_" />
+                    </Typography.Heading>
+                    <Typography.Paragraph size={is_mobile_or_tablet ? 'large' : 'medium'}>
```

Comment:
By new approach we always should pass only 1 parameter. If design request exactly this, you need to let them know that it's theirs mistake. Feel free to use our squad channel